README.md

```diff
@@ -225,6 +225,60 @@ After this is complete, Grafana will show two new dashboards: `Kafka Metrics` an
 ## Security
 Security issues in the Charmed Kafka Operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.
 
+## Performance Tuning
+#### Virtual Memory Handling - Recommended
+Kafka brokers make heavy use of the OS page cache to maintain performance. They never normally explicitly issue a command to ensure messages have been persisted to disk (`sync`), relying instead on the underlying OS to ensure that larger chunks (pages) of data are persisted from the page cache to the disk when the OS deems it efficient and/or necessary to do so. As such, there are a range of runtime kernel parameter tuning that are recommended to set on machines running Kafka to improve performance.
```

Comment:
**question:** I'm green on these details. Is this affecting the machine where the charm is running or the host that spawns the units?

Reply:
Machine running the charm. 
These checks essentially say 'hey, your machine might need reconfiguring as you're going to hit limits soon'.

Reply:
If these checks fail, we'll set a log entry and a status. Have a look at the `literals.py` diff for them. Does that cover what you mean?